### DISCUSSION TITLE
Configuring SkyWalking to Store Data in a Secure Elasticsearch Database

### DISCUSSION THREAD

**Discussion by abhishekacharya828** [apache/skywalking#12123] [General]
In the documentation, it is specified that only insecure Elasticsearch can be used to store data. However, I want to store data in a secure Elasticsearch database. When I change the protocol from HTTP to HTTPS protocol , it throws a configuration error. How can I store data with a secure Elasticsearch database ?

![Screenshot from 2024-04-17 12-36-28](https://github.com/apache/skywalking/assets/134144944/e9e6a428-8152-436a-a49d-a0b7081c7215)

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/main/latest/en/setup/backend/storages/elasticsearch/#elasticsearch-with-https-ssl-encrypting-communications

Please note this, you may miss this
- protocol: ${SW_STORAGE_ES_HTTP_PROTOCOL:"https"}

**Reply by abhishekacharya828** [reply]
please check properly i wrote this line in protocol section

**Reply by wu-sheng** [reply]
I can't see your error, don't know what to check.
